1.0/docs/start/first-element/step-2.md

```diff
@@ -0,0 +1,255 @@
+---
+layout: default
+type: start
+shortname: Start
+title: "Step 2: Add Local DOM"
+subtitle: Build your first Polymer element
+---
+
+<style>
+.caption + pre,.caption + .prettyprint {
+  margin-top: 0px;
+}
+</style>
+
+Next, you'll create a simple element that displays an icon.
+
+In this step, you'll learn about:
+
+*   Creating a custom element using Polymer.
+*   Working with local DOM.
+
+
+<div class="alert alert-info"><strong>Learn more: local DOM.</strong> Local DOM
```

Comment:
Hm. I feel like I have to have something here to introduce the term. Do you think I should mention shadow DOM? I wanted to avoid clouding the waters, but as you said, people may already have heard of it.